lingui.config.ts

```diff
@@ -1,18 +1,14 @@
 /** @type {import('@lingui/conf').LinguiConfig} */
 module.exports = {
-  locales: ['en', 'es', 'jp', 'fr'],
+  locales: ['en', 'es', 'ja', 'fr'],
```

Comment:
Why we need this change?

Reply:
Because 'ja' is the standard of Japanese abbreviation

Reply:
I'm not sure, what is your reference? 
I checked these sources:
https://www.ncbi.nlm.nih.gov/books/NBK7249/
https://www.loc.gov/standards/iso639-2/php/code_list.php

Reply:
according to the crowdin refrence:
https://developer.crowdin.com/language-codes/

Also there is 'ja' in this link that you sent:
https://www.loc.gov/standards/iso639-2/php/code_list.php

Reply:
thanks. please add [this link](https://www.loc.gov/standards/iso639-2/php/code_list.php) and mention we are using **ISO 639-1 Code** as well.

Reply:
Done